src/content/pages/faq.mdx

```diff
@@ -45,6 +45,19 @@ A. The conference will be organised in three phases:
 2. Wednesday - Friday (10.07 - 12.07): main conference days
 3. Saturday & Sunday (13.07 & 14.07): sprint days
 
+
+## **Q. Will there be food provided at the conference?**
+
+A. Yes, we will provide a light lunch and coffee breaks during the conference days. We will also have a social event on one of the evenings, where you can enjoy food and drinks with your fellow attendees.
```

Comment:
Might be good to mention that social event is not free? Or we can even skip this part? WDYT?

Reply:
I thought it might be better to skip it. For people who have booked to go to the social event, they would refer to the social event page for the entire programme and food information. Here, might be good to just mention the general conf info?

Reply:
Will remove.